WordPress/src/main/java/org/wordpress/android/ui/notifications/adapters/NotesAdapter.java

```diff
@@ -296,13 +298,13 @@ public void onBindViewHolder(NoteViewHolder noteViewHolder, int position) {
         boolean isUnread = note.isUnread();
 
         int gridicon = mNoticonUtils.noticonToGridicon(note.getNoticonCharacter());
-        mImageManager.load(noteViewHolder.mNoteIcon, gridicon);
+        noteViewHolder.mImgAvatar.setBadgeIcon(gridicon);
         if (commentStatus == CommentStatus.UNAPPROVED) {
-            noteViewHolder.mNoteIcon.setBackgroundResource(R.drawable.bg_oval_warning_stroke_surface);
+            noteViewHolder.mImgAvatar.setBadgeBackground(R.drawable.bg_oval_warning);
```

Comment:
We should probably rename `bg_oval_warning` to `bg_oval_warning_dark` since there is also a light variation of the warning color :)

Reply:
Good catch! Will update it.